src/components/Subscribe/Subscribe.tsx

```diff
@@ -33,6 +33,22 @@ export interface SubscribeProps extends React.HTMLAttributes<HTMLFormElement> {
    * Subscribe title text
    */
   title?: string;
+  /**
+   * Subscribe loading text
+   */
+  loadingText?: string;
```

Comment:
maybe should make this a ReactNode just in case...

Reply:
Don't think so cause if we have to add spinner or something spinner won't be on that place, so we'll have to change our component anyway